lib/routes/stheadline/std/realtime.ts

```diff
@@ -20,14 +19,13 @@ async function handler(ctx) {
     const { data: response } = await got(url);
     const $ = load(response);
 
-    let items = $(`${category === '即時' ? '.moreNews > .col-md-4' : ''} .media-body > .my-2 > a`)
+    let items = $(`.news-block .news-detail > a`)
         .toArray()
         .map((item) => {
             item = $(item);
             return {
-                title: item.attr('title'),
-                link: item.attr('href'),
-                guid: item.attr('href').slice(0, item.attr('href').lastIndexOf('/')),
```

Comment:
This changes the GUID which causes duplicated articles appears in users' RSS reader. Apart from that, including the title in GUID means if the site changes the title of an article multiple times, the article will reappear in users' RSS reader.